Chainguard Libraries for Python docs draft

smythp opened the pull request:
## Description

Getting the ball rolling on our docs for Chainguard Libraries. This PR includes a draft of the overview page. I'll add to these docs as I learn more. Currently, the production artifacts are being pulled down and rebuilt and there are a number of unknowns as to URL structure and etc.

Update: added draft for global configuration (repository manager configuration).

## Notes

References https://github.com/chainguard-dev/internal/issues/4839

smythp committed:
Overview page for Chainguard Libraries for Python

Signed-off-by: Patrick Smyth <patrick.smyth@chainguard.dev>

smythp committed:
Structure for additional Python pages

Signed-off-by: Patrick Smyth <patrick.smyth@chainguard.dev>

smythp committed:
Adapted Cloudsmith instructions

Signed-off-by: Patrick Smyth <patrick.smyth@chainguard.dev>

smythp committed:
Added Chainguard Libraries for Python global configuration file

Signed-off-by: Patrick Smyth <patrick.smyth@chainguard.dev>

mosabua commented on content/chainguard/libraries/java/global-configuration.md:
```diff
@@ -23,7 +23,7 @@ Repository](https://www.sonatype.com/products/sonatype-nexus-repository). The
 development tools to retrieve the required libraries.
 
-At a high level, adopting the use of Chainguard Libraries consists of the the
+At a high level, adopting the use of Chainguard Libraries consists of the
```
Move this into a separate PR to avoid conflicts.. also .. might be fixed already in main

smythp committed:
Reverted changes outside the Python folder

mosabua commented on content/chainguard/libraries/python/management.md:
```diff
@@ -0,0 +1,68 @@
+
+## Increase Chainguard Library Use
+
+The number of available artifacts in Chainguard Libraries for Java increases
```
```suggestion
The number of available artifacts in Chainguard Libraries for Python increases
```

mosabua commented on content/chainguard/libraries/python/management.md:
```diff
@@ -0,0 +1,68 @@
+## Increase Chainguard Library Use
+
+The number of available artifacts in Chainguard Libraries for Java increases
+over time. If an artifact was already retrieved from the Maven Central
```
```suggestion
over time. If an artifact was already retrieved from the PyPI
```

mosabua commented on content/chainguard/libraries/python/management.md:
```diff
@@ -0,0 +1,68 @@
+not automatically replaced with the equivalent Chainguard Library version. 
+
+You can force a download of new libraries by erasing them from your local
+repositories on your workstations and the Maven Central proxy repository in your
```
```suggestion
repositories on your workstations and the PyPI proxy repository in your
```

mosabua commented on content/chainguard/libraries/python/management.md:
```diff
@@ -0,0 +1,68 @@
+
+After the deletion any new build retrieves the artifact again and attempts to
+download from the Chainguard repository. As a result, newly available artifacts
+replace old artifacts that originated from Maven Central and your use of
```
```suggestion
replace old artifacts that originated from PyPI and your use of
```

mosabua reviewed (changes requested):
Looking good already .. once we can confirm this all working we are in a good position to merge as needed .. and we can keep iterating as well

smythp committed:
Added build configuration

Moved accessing credentials from global to build file

Added basic starter info on pip and uv

Signed-off-by: Patrick Smyth <patrick.smyth@chainguard.dev>

smythp committed:
Removed old references to maven URL structure

Signed-off-by: Patrick Smyth <patrick.smyth@chainguard.dev>

mosabua commented on content/chainguard/libraries/python/overview.md:
```diff
@@ -0,0 +1,70 @@
+
+## Introduction
+
+Python is the second most popular programming language in the world and the open [Python Package Index (PyPI)](https://pypi.org/) contains over 600,000 libraries for application development, machine learning, data science, and many other use cases. Chainguard Libraries for Python rebuilds these powerful open source projects within the Chainguard Factory, enabling access to the Python ecosystem wh ...
```
I just think that its safer to not claim a specific position .. it changes with what survey and data you look at .. alternative would "one of the most popular programming languages..."

smythp committed:
Updates from Manfred's review

Signed-off-by: Patrick Smyth <patrick.smyth@chainguard.dev>

smythp committed:
Updated Python popularity language

Signed-off-by: Patrick Smyth <patrick.smyth@chainguard.dev>

smythp committed:
Language updates based on Manfred's review

Signed-off-by: Patrick Smyth <patrick.smyth@chainguard.dev>

smythp committed:
Apply suggestions from code review

Co-authored-by: Manfred Moser <manfred@simpligility.ca>
Signed-off-by: Patrick Smyth <patrick.smyth@chainguard.dev>

mosabua reviewed (approved):
Looks good to me for launch. We fully expect to refine further.
